Assets/HoloToolkit/Input/Scripts/InputSources/EditorInputSource.cs

```diff
@@ -468,29 +473,30 @@ private void SendControllerStateEvents(float time)
             {
                 if (CurrentButtonStates.IsMenuButtonDown)
                 {
-                    InputManager.Instance.RaiseSourceDown(this, controllerId, InteractionSourcePressType.Menu);
+                    InputManager.Instance.RaiseSourceDown(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Menu);
                 }
                 else
                 {
-                    InputManager.Instance.RaiseSourceUp(this, controllerId, InteractionSourcePressType.Menu);
+                    InputManager.Instance.RaiseSourceUp(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Menu);
                 }
             }
 
             if (CurrentButtonStates.GraspStateChanged)
             {
                 if (CurrentButtonStates.IsGrasped)
                 {
-                    InputManager.Instance.RaiseSourceDown(this, controllerId, InteractionSourcePressType.Grasp);
+                    InputManager.Instance.RaiseSourceDown(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Grasp);
                 }
                 else
                 {
-                    InputManager.Instance.RaiseSourceUp(this, controllerId, InteractionSourcePressType.Grasp);
+                    InputManager.Instance.RaiseSourceUp(this, controllerId, (InteractionSourcePressInfo)InteractionSourcePressType.Grasp);
                 }
             }
+#endif
         }
 
         /// <summary>
-        /// Sends the events for hand visibility changes & controller connect/disocnnect.
+        /// Sends the events for hand visibility changes &amp; controller connect/disconnect.
```

Comment:
Is `&amp;` actually what you wanted here?

Reply:
Yeah https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1028#discussion_r142283329